README.md

```diff
@@ -44,14 +44,14 @@ The ReactOS project, although currently focused on Windows Server 2003 compatibi
 
 The code of ReactOS is licensed under [GNU GPL 2.0](https://github.com/reactos/reactos/blob/master/COPYING).
 
-## Building 
+## Building
 
 [![appveyor.badge]][appveyor.link] [![travis.badge]][travis.link] [![rosbewin.badge]][rosbewin.link] [![rosbeunix.badge]][rosbeunix.link] [![coverity.badge]][coverity.link]
 
-To build the system it is strongly advised to use the _ReactOS Build Environment (RosBE)._ 
-Up-to-date versions for Windows and for Unix/GNU-Linux are available from our download page at: http://www.reactos.org/wiki/Build_Environment.
+To build the system it is strongly advised to use the _ReactOS Build Environment (RosBE)._
+Up-to-date versions for Windows and for Unix/GNU-Linux are available from our download page at: [http://www.reactos.org/wiki/Build_Environment](Build Environment).
```

Comment:
Checking the other examples, you can see it's the other way around: `[Human-readable description](http://www.url.link)`.

Reply:
Maybe also enclose the human-readable description if it stands on itself (as it is the case here: "Build Environment")